packages/gatsby-source-wordpress/src/steps/source-nodes/fetch-nodes/fetch-referenced-media-items.js

```diff
@@ -361,7 +361,7 @@ export const fetchMediaItemsBySourceUrl = async ({
   // so we need to resolve this promise
   // otherwise it will never resolve below.
   if (!mediaItemUrlsPages.length) {
-    return Promise.resolve([])
+    return Promise.resolve(previouslyCachedMediaItemNodes)
   }
```

Comment:
Interesting, so we we were getting the `previouslyCachedMediaItemNodes` already but not using it.

Reply:
Yeah 😅 must've been a mistake in another PR because the var was there in scope but not being used anywhere. The new test will catch that kind of mistake so we should be good going forward